core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/jdbc/JDBCUserStoreManager.java

```diff
@@ -3930,7 +3930,8 @@ public long doCountUsersWithClaims(String claimUri, String value) throws UserSto
         String sqlStmt;
         if (isUserNameClaim(claimUri)) {
             sqlStmt = realmConfig.getUserStoreProperty(JDBCRealmConstants.COUNT_USERS);
-
+        } else if (ExpressionOperation.EQ.toString().toLowerCase().equals(valueFilter)) {
```

Comment:
Don't we need to compare this with ignore case?

Reply:
Addressed it with this commit https://github.com/wso2/carbon-kernel/pull/3642/commits/824fe0381e39b6d89340388f531cccab35f657a1